samples/ComputerVision/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,227 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace AnalyzeImage
+{
+    class AnalyzeImage
+    {
+        private const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Service subscription key here
+
+        // Specify the features to return
+        private static readonly List<VisualFeatureTypes> features =
+            new List<VisualFeatureTypes>()
+            {
+            VisualFeatureTypes.Categories, VisualFeatureTypes.Description,
+            VisualFeatureTypes.Faces, VisualFeatureTypes.ImageType,
+            VisualFeatureTypes.Tags, VisualFeatureTypes.Adult,
+            VisualFeatureTypes.Color, VisualFeatureTypes.Brands,
+            VisualFeatureTypes.Objects
+            };
+
+        static void Main(string[] args)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(subscriptionKey));
+
+            // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+            computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com";
+
+            // localImagePath = @"C:\Documents\LocalImage.jpg"
```

Comment:
Let's update these two lines to say this. 
// See repo readme.md for info on how to get these images. Alternatively, you can just set it to an image on your machine.
string localImagePath = @"celebrities.jpg"; 

Notice tat I removed the Images folder from the path. That way people can just add a single image per project (since each project just needs one image anyway). Please propagate this to all cases so we are consistent.